lib/Plerd.pm

```diff
@@ -913,6 +918,21 @@ the blog's docroot -- in other words, the place Plerd will write HTML and XML fi
 A L<Path::Class::Dir> object representation of the directory that holds
 the blogs's private, not-necessarily-human-readable data files.
 
+=item tags_publication_directory
+
+A L<Path::Class::Dir> object representation of the directory within the docroot that holds
+tag index HTML files.
+
+=item tag_index_uri
+
+This is a L<URI> object that points to the tag index.  It is
+particularly helpful when creating navigation.
+
+=item tag_uri TAG
```

Comment:
Please move this to the "OBJECT METHODS" list instead, and format the argument list per the other argument-taking methods found there.